### ISSUE TITLE
Expand/ Collapse All button could be more descriptive

### ISSUE THREAD

**Issue by ferBonnin** [microsoft/accessibility-insights-web#3539]
## Describe the bug

For screen reader users, the first time they explore the detailed results the Expand / Collapse all button is confusing since there is no context (until there is more tabbing) on what the button is acting on.

## Expected behavior
A aria-label or aria-description that indicates that the button is expanding **all failure instances**

## Context (please complete the following information)

- OS Name & Version:  Windows
- AI-Web Version & Environment:  Accessibility Insights for Web and for Android
## Additional context

From user studies feedback

**Comment by LiLoDavis** [comment]
@ferBonnin This is in FastPass > Automated checks?

**Comment by ferBonnin** [comment]
> @ferBonnin This is in FastPass > Automated checks?

yes in both our Web and Android tools

**Comment by LiLoDavis** [comment]
An `aria-label` attribute could communicate the function more clearly without changing the visible text. Something like this: `aria-label="Expand all rules to see failed instances."` For screen reader users, the `aria-label` would override the screen text.

**Comment by DaveTryon** [comment]
@ferBonnin, this is fixed in the current Canary bits. Please validate/close when convenient. Thanks!

**Comment by ferBonnin** [comment]
validated in Canary with NVDA, thanks!
